src/core/server/metrics/collectors/process.ts

```diff
@@ -25,19 +37,17 @@ export class ProcessMetricsCollector implements MetricsCollector<OpsProcessMetri
         resident_set_size_in_bytes: memoryUsage.rss,
       },
       pid: process.pid,
-      event_loop_delay: eventLoopDelay,
+      event_loop_delay: eventLoopDelayHistogram.mean,
+      event_loop_delay_histogram: eventLoopDelayHistogram,
       uptime_in_millis: process.uptime() * 1000,
     };
   }
 
-  public reset() {}
-}
+  public collect(): OpsProcessMetrics[] {
+    return [this.getCurrentPidMetrics()];
+  }
```

Comment:
(just thinking out loud, please ignore) I know this array format is better for the downstream indexation, but I'm still not a big fan compared to a Record<workerId, Data>